docs/README-windows.md

```diff
@@ -16,6 +16,68 @@ Studio solution.
 
 Details are here: https://github.com/libsdl-org/SDL/issues/5186
 
+## MinGW-64 compiler support
+
+SDL can be built with MinGW-64 and CMake. First, you need to install and set up the MSYS2 environment, which provides the MinGW-64 toolchain. Install MSYS2, typically to `C:\msys64`, and follow the instructions on the MSYS2 wiki to use the MinGW-64 shell to update all components in the MSYS2 environment. This generally amounts to running `pacman -Syuu` from the mingw64 shell, but refer to MSYS2's documentation for more details. Once the MSYS2 environment has been updated, install the x86_64 MinGW toolchain from the mingw64 shell with the command `pacman -S mingw-w64-x86_64-toolchain`. (You could alternatively install the 32-bit MinGW toolchain from the mingw32 shell.)
+
+To build and install SDL, you can use PowerShell or any CMake-compatible IDE. First, install CMake, Ninja, and Git. These tools can be installed using the Windows Package Manager `winget` (The winget IDs for these tools are Kitware.CMake, Git.Git, and Ninja-build.Ninja). Clone SDL to an appropriate location with `git` and run the following commands from the root of the cloned repository:
```

Comment:
Adding names and instructions for specific packager tools is a slippery slope. 
There exist alternative and more lightweight toolchains (e.g. [w64devkit](https://github.com/skeeto/w64devkit/releases/tag/v2.0.0)) that only package the compiler and c library.

I think it's worth to clarify that you can get your tools from various sources (msys2, winget, chocolatey, ...), but building SDL will still follow the same pattern. The one you documented are fine. I just want to avoid receiving prs adding documentation for yet another mingw toolchain.